architecture/openshift_model.adoc

```diff
@@ -39,10 +39,13 @@ Source-to-image (sti) is a tool for building reproducible Docker images. It prod
 The custom build strategy is very similar to *Docker build* strategy, but users might customize the builder image that will be used for build execution. The Docker build uses https://registry.hub.docker.com/u/openshift/docker-builder/[openshift/docker-builder] image by default. Using your own builder image allows you to customize your build process.
 
 == BuildLog
+A buildlog is a model object that stores the logs of a particular build for later inspection.
 
 == Deployment
+A deployment is a specially annotated replication controller, specifying the desired configuration of that controller. See the https://github.com/openshift/origin/blob/master/docs/deployments.md[deployment documentation] for more information.
```

Comment:
I assume the long-term plan is not to leave this pointing at dev docs :)

FWIW, I think the idea of deployments is probably stable enough to begin documenting if you want to bring that over to openshift-docs so you have a reference for this to point at.